api/controller/ReportController_comingsoon.py

```diff
@@ -0,0 +1,231 @@
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(2)
+                                },
+                                "three": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(3, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(3)
+                                },
+                                "four": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(4, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(4)
+                                },
+                                "five": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(5, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(5)
+                                },
+                                "six": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(6, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(6)
+                                },
+                                "seven": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(7, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(7)
+                                },
+                                "eight": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(8, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(8)
+                                },
+                                "nine": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(9, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(9)
+                                },
+                                "ten": {
+                                    "monthly_savings": report.downpayment_goal_monthly_savings(10, report.downpayment_savings, report.downpayment_percentage),
+                                    "goal_end_date": report.downpayment_savings_goal_end_date(10)
+                                },
+                            }
+                        }
+                    }
+                }
+            }
+        }, 200
+
```

Comment:
We are one phase away from making this possible!

Reply:
Awesome! What phrase?

Reply:
Lol!  I was just commenting on how excited I was to see this coming to life.